configure.py

```diff
@@ -113,6 +128,9 @@ def _get_external_deps(args):
         deps.append("ffmpeg")
         deps.append("sdl2")
         deps.append("glslang")
+        deps.append("harfbuzz")
+        deps.append("freetype")
+        deps.append("fribidi")
```

Comment:
nit: freetype, harfbuzz, fribidi for consistency with the other order you picked?

Reply:
Done.